source/quick-start.txt

```diff
@@ -0,0 +1,136 @@
+.. _entity-framework-quickstart:
+
+===========
+Quick Start
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/quick-start/overview.rst
+
+Create a MongoDB Cluster
+------------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Set Up a Free Tier Cluster in Atlas
+
+      To set up your Atlas free cluster required for this Quick Start, complete the guidei
+      on :guides:`MongoDB Atlas Setup </atlas/account>`.
+
+      After completing the steps in the Atlas guide, you have a new MongoDB
+      cluster deployed in Atlas, a new database user, and
+      sample datasets loaded into your cluster. You also have
+      a connection string similar to the following in your copy buffer:
+
+      .. code-block:: bash
+         :copyable: false
+
+         "mongodb+srv://<username>:<password>@cluster0.abc.mongodb.net/?retryWrites=true&w=majority"
+
+   .. step:: Update the Placeholders
+
+     Paste the connection string in your copy buffer into a file in your preferred text
+     editor. Replace the ``<username>`` and ``<password>`` placeholders with
+     your database user's username and password.
+
+     Save this file to a safe location for use in the next step.
+
+   .. step:: Set Your Connection String
+
+      Run the following code in your shell to save the MongoDB connection string
+      in your copy buffer from the previous step to an 
+      environment variable. This method is safer than including your credentials in your source 
+      code. 
+
+   .. code-block:: bash
+
+      export MONGODB_URI='<your connection string>'
+
+   .. important::
+
+      Make sure to replace the ``<username>`` and ``<password>`` sections of the connection 
+      string with the username and password of your Atlas user.
+
+Set Up Your Project
+-------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Project
+
+      Create a new directory and initialize your project with the ``dotnet new`` command,
+      as follows:
+
+      .. code-block:: shell
+
+         mkdir entity-quickstart
+         cd entity-quickstart
+         dotnet new console
+
+   .. _entity-framework-add-mongodb-dependency:
+
+   .. step:: Add the {+provider-long+} as a Dependency
+
+      Use the ``dotnet add`` command to add the {+provider-short+} to your project as a
+      dependency.
+
+      .. code-block:: shell
+
+         dotnet add package {+package+} 
+
+Query Your MongoDB Cluster from Your Application
+------------------------------------------------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Code File
```

Comment:
Suggestion:

I think this task includes more than creating a file since it includes adding sample code to it. Perhaps a more descriptive name could be "Add the Sample Code" or something similar.